### PR TITLE
ImageClassificationMobileNetV2.ipynb added

### PR DESCRIPTION
**Pull request by JayShimpi22**

### CODE REVIEW

**Comment by Thilakraj1998** [comment]
required changes:
1. add GPU required tag(html code available in CNN file) in the meta description cell
2.  your output layer of neural network has an issue, number of neurons in last layer should be equal to number of target class. In your application case it should be 5.
3. Your sample images only have a single class representation

**Comment by Thilakraj1998** [comment]
in case of binary classification keep your loss funciton as:
loss = "binary_crossentropy" 

and final layer has
model.add(layers.Dense(1, activation=output_activation))

also avoid this function def get_confusion_matrix(data, N):

if possible share the colab file with me

**Comment by JayShimpi22** [comment]
okay sir, I will fix it

**Comment by JayShimpi22** [comment]
Updated File

**Comment by Thilakraj1998** [comment]
> Updated File

Also change activation function at last layer to sigmoid

**Comment by JayShimpi22** [comment]
Done sir! Thanks for info.

**Comment by JayShimpi22** [comment]
Done sir! Thanks for info.
